examples/track.rs

```diff
@@ -57,13 +86,20 @@ fn main() {
 
     dispatcher.setup(&mut world.res);
 
-    for _ in 0..10000 {
+    for _ in 0..50 {
```

Comment:
Was this reduced because it was too slow?

Reply:
no, mainly for readability